common/services/move.js

```diff
@@ -38,6 +38,7 @@ const defaultInclude = [
 ]
 
 function getAll({
+  apiClient,
```

Comment:
This is outside of the service scope and doesn't have access to the req. It could be moved inside in which case this passing of apiClient won't be necessary, but kept outside for minimum changes. Happy to revisit.

Reply:
It think it would be a good idea to move it inside and to perhaps change the name to something less confusing than the duped `getAll`

Previously the biggest problem we had with service calls was batched requests overwhelming the BE - 
If we don't "pass" the req somehow, all the calls that end up using it won't be able to send the transactionId or user details or (future handwavey) do useful logging.

Reply:
But maybe you're right. Maybe the correct solution is to get to the point where the batching can simply be removed.

Reply:
There is a comment in the batch file about removing it once services know about a user's locations, which after this refactor, they will be able to, so there's probably a case for tackling that properly soon after this work is complete.